Sources/StreamVideo/OpenApi/generated/Models/RecordSettingsRequest.swift

```diff
@@ -1,71 +1,53 @@
 //
-// RecordSettingsRequest.swift
-//
-// Generated by openapi-generator
-// https://openapi-generator.tech
+// Copyright © 2024 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
-
-
-public struct RecordSettingsRequest: Codable, JSONEncodable, Hashable {
+    
+public final class RecordSettingsRequest: @unchecked Sendable, Codable, JSONEncodable, Hashable {
+    
     public enum Mode: String, Codable, CaseIterable {
-        case available = "available"
-        case disabled = "disabled"
         case autoOn = "auto-on"
+        case available
+        case disabled
         case unknown = "_unknown"
 
         public init(from decoder: Decoder) throws {
             let container = try decoder.singleValueContainer()
             if let decodedValue = try? container.decode(String.self),
-                let value = Self(rawValue: decodedValue) {
-                self = value
-            } else {
-                self = .unknown
-            }
-        }
-    }
-    public enum Quality: String, Codable, CaseIterable {
```

Comment:
Why is this one removed?

Reply:
Should the embedded enums be `Sendable` as well?